packages/frontend/src/features/sqlRunner/components/BarChartFieldConfiguration.tsx

```diff
@@ -149,6 +191,13 @@ export const BarChartFieldConfiguration = () => {
     const series = useAppSelector(
         (state) => state.barChartConfig.config?.display?.series,
     );
+    const groupByFields = useAppSelector(
+        (state) => state.barChartConfig.config?.fieldConfig?.groupBy,
+    );
+    const groupByField = groupByFields?.[0];
```

Comment:
Could you just select this directly or do you need groupByFields for something else?

Reply:
Fixed!